Vision/Vision.bs

```diff
@@ -160,8 +160,10 @@ Markup Shorthands: markdown yes
 		**Free to Implement**: Our standards are rooted in a strong royalty-free patent policy
 		and open copyright licenses.
 	<li id=voluntary-impl>
-		**Voluntary to Implement**: Our standards span many diverse use-cases and industries.
-		The success of our standards is determined 
+		**Voluntary to Implement**: Our work spans many diverse use-cases and industries,
+		and our standards matter when implementers choose to adopt them. 
+		Rather than constraining anyone to use our designs, 
+		the success of our standards is determined 
```

Comment:
> and our standards matter when implementers choose to adopt them. 

is basically redundant with

> the success of our standards is determined by the market, through voluntary adoption across a broad set of stakeholders.

and

> Rather than constraining anyone to use our designs, 

might be somewhat clarifying by providing an idea that contrasts with "voluntary", it doesn't actually help connect the two thoughts in the way you expressed in the issue.

So overall... I don't think this actually solves the problem of [expressing that connection](https://github.com/w3c/AB-public/issues/248#issuecomment-2707273692).